https://bugs.gentoo.org/829134
https://invent.kde.org/sdk/clazy/-/commit/0ee1ed9ff1bd4cf2a49e1232696747f1898e2987.patch

From: Allen Winter <allen.winter@kdab.com>
Date: Sat, 11 Dec 2021 14:57:52 -0500
Subject: [PATCH] src/SourceCompatibilityHelpers.h - fix compile with LLVM<12

fix the GET_LEXER macro for LLVM versions less than 12
--- a/src/SourceCompatibilityHelpers.h
+++ b/src/SourceCompatibilityHelpers.h
@@ -123,7 +123,7 @@ clang::Lexer(id, inputFile.getValue(), sm, lo)
 
 #else
 #define GET_LEXER(id, inputFile, sm, lo) \
-clang::Lexer(id, inputFile.getValue(), sm, lo)
+clang::Lexer(id, inputFile, sm, lo)
 #endif
 
 inline bool isFinal(const clang::CXXRecordDecl *record)
GitLab
